docker-compose/monitor/docker-compose-connector.yml

```diff
@@ -3,21 +3,26 @@ services:
   jaeger:
     networks:
       - backend
-    image: jaegertracing/all-in-one:latest
+    image: jaegertracing/all-in-one:dev
```

Comment:
Maybe parameterize to allow running without rebuilding.

Reply:
Okay, I've parameterised this, which is set in the Makefile in: [ba0a3d7](https://github.com/jaegertracing/jaeger/pull/4452/commits/ba0a3d759c91ad1ef6e63da110e509c42eaad9ef). 

Let me know if that addresses your comment and, if not, can you please clarify what you mean?